src/nvc.c

```diff
@@ -291,10 +252,27 @@ static int elaborate(int argc, char **argv)
          opt_set_str(OPT_DUMP_VCODE, optarg ?: "");
          break;
       case 'c':
-         if (optarg)
-            cover_mask = parse_cover_mask(optarg);
+         if (optarg) {
+            char LOCAL *wc_buf[1] = {
```

Comment:
This actually does the equivalent of `free(wc_buf)` instead of `free(wc_buf[0])` which is presumably what you want. But see my comment above: this code would be much less convoluted if we just had a single dedicated function for parsing the coverage options.

Reply:
I reworked the `wc_buf` to be part of `opt_separed_t`. This fixes the issue, leaves no leak, and
if there are multiple wild-card commands, its easy to find wild-card for each command.